.eslintrc.js

```diff
@@ -13,9 +14,9 @@ module.exports = {
 		wpApiSettings: true,
 		eejsdata: true
 	},
-	plugins: [
-		'jest',
-	],
+	rules: {
+		"linebreak-style": 0
+	},
```

Comment:
I know your commit message said you added this linebreak-style for windows but I'm wondering _why_ you are adding this?  We've imported the `eslint-config-wordpress` ruleset which has this:

```
// Disallow mixed "LF" and "CRLF" as linebreaks
'linebreak-style': ['error', 'unix'],
```

In other words we're INTENTIONALLY avoiding CRLF in our code.   So I'd prefer you did not override this rule.  If you're getting linting fails locally, that's something you need to fix locally as the linter is performing as expected.  There's many options that you can use to do so, one of them is implement in phpStorm the reading of our  `.editorconfig` file.  Here's a github issue that may be useful to you as well: https://github.com/diegohaz/arc/issues/171

Reply:
I also know your rule just disables the linebreak style checks.  I'd still prefer if you left the rules as is.

Reply:
I really don't know what to say then, because what you are seeing does not jive with either what I see locally or what travis sees.

Reply:
as a short term fix for your needs locally, you can run just the js unit-tests without a lint check via `npm run test-unit`.   However, long-term you'll want to look into what is changing the line endings on your machine.